book/src/configuration.md

```diff
@@ -307,4 +308,17 @@ Currently unused
 
 #### `[editor.gutters.spacer]` Section
 
-Currently unused
\ No newline at end of file
+Currently unused
+
+### `[editor.icons]` Section
+
+Option for displaying icons within the editor.
+
+> Warning: File-type and symbol-kind icons require a patched font such as [NerdFonts](https://www.nerdfonts.com/) to be installed and configured in your terminal emulator.
```

Comment:
I've changed this line to:

> Warning: some symbols (such as file-type and symbol-kind icons that you would see in the picker) are not available in the "default" icon set. They usually require a patched font such as [NerdFonts](https://www.nerdfonts.com/) to be installed and configured in your terminal emulator, and the corresponding icon set to be configured in the editor (for example, using `icons = "nerdfonts"` in your configuration file).

Although, I don't really see how file-type icons could be made out of unicode characters since they're mostly language logos.